src/cljs/reagent_toolbox_docs/components/dropdown.cljs

```diff
@@ -0,0 +1,167 @@
+;;;; Copyright © 2017 Flexpoint Tech Ltd
+
+(ns reagent-toolbox-docs.components.dropdown
+  (:require [reagent-toolbox-docs.layout :as layout]
+            [reagent-toolbox-docs.ui :as ui]))
+
+(defn view
+  []
+  (fn []
+    [:article
+     [:h1 "Dropdown"]
+     [:p "The Dropdown selects an option between multiple selections. The element displays "
+      "the current state and a down arrow. When it is clicked, it displays the list of "
+      "available options."]
+
+     [ui/display-and-eval-code
+      "(defn dropdown-test
+         []
+         (let [countries [{:value \"EN-gb\" :label \"England\"}
+                          {:value \"ES-es\" :label \"Spain\"}
+                          {:value \"TH-th\" :label \"Thailand\"}
+                          {:value \"EN-en\" :label \"USA\"}]
+               state (reagent.ratom/atom {:value \"ES-es\"})]
+           (fn []
+             [:div
+              [:p \"Value: \" (pr-str @state)]
+              [reagent-toolbox.core/dropdown {:auto true
+                                              :on-change #(swap! state assoc :value %)
+                                              :source countries
+                                              :value (:value @state)}]])))
+
+       [dropdown-test]"]
+
+     [:p "If you want to provide a theme via context, the component key is RTDropdown."]
+
+     [:section
+      [:h2 "Properties"]
+
+      [:table
+       [:thead
+        [:tr
+         [:th "Name"]
+         [:th "Type"]
+         [:th "Default"]
+         [:th "Description"]]]
+       [:tbody
+        [:tr
+         [:td [:code "allow-blank"]]
+         [:td [:code "boolean"]]
+         [:td [:code "true"]]
+         [:td "If false the dropdown will preselect the first item if the supplied value "
+          "matches none of the options' values."]]
```

Comment:
For example, in this case, I would put them in one line, but break them at 120 following this exact method of breaking them (with the space on the top line).